fastlane/Deliverfile

```diff
@@ -11,7 +11,7 @@ screenshots_path "./screenshots/"
 app_identifier "com.automattic.SimplenoteMac"
 
 # Make sure to update these keys for a new version
-app_version "1.11.1"
+app_version "2.2"
```

Comment:
So... the release script didn't mention to run the `code_freeze` lane and I never thought of it either 😳 (_I feel very silly for that_). This time around, it occurred to me, and so we have this non linear change in the file.

I've updated the script in a draft PR on its repository.